Content.Shared/Silicons/StationAi/SharedStationAiSystem.cs

```diff
@@ -34,13 +38,18 @@ public abstract partial class SharedStationAiSystem : EntitySystem
     [Dependency] private   readonly SharedAppearanceSystem _appearance = default!;
     [Dependency] private   readonly SharedContainerSystem _containers = default!;
     [Dependency] private   readonly SharedDoorSystem _doors = default!;
+    [Dependency] private   readonly SharedElectrocutionSystem _electrify = default!;
+    [Dependency] private   readonly SharedAirlockSystem _airlocks = default!;
     [Dependency] private   readonly SharedEyeSystem _eye = default!;
     [Dependency] protected readonly SharedMapSystem Maps = default!;
     [Dependency] private readonly SharedMindSystem _mind = default!;
     [Dependency] private   readonly SharedMoverController _mover = default!;
     [Dependency] private   readonly SharedTransformSystem _xforms = default!;
     [Dependency] private   readonly SharedUserInterfaceSystem _uiSystem = default!;
     [Dependency] private   readonly StationAiVisionSystem _vision = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+    [Dependency] protected readonly SharedPowerReceiverSystem PowerReceiver = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
```

Comment:
Alphabetical and follow format of the existing stuff please I beg.

Reply:
fine, but it was not me who started it (39 line by original file)

Reply:
not 100% sure what u wanted but i sorted it all by field names.